### DISCUSSION TITLE
What is the different between typical and initial

### DISCUSSION THREAD

**Discussion by loveinSH** [pybamm-team/PyBaMM#3587] [Q&A]
In code lithium_ion_parameters there is:

# Lithium plating parameters
        self.V_bar_Li = pybamm.Parameter(
            "Lithium metal partial molar volume [m3.mol-1]"
        )
        self.c_Li_typ = pybamm.Parameter(
            "Typical plated lithium concentration [mol.m-3]"
        )
        self.c_plated_Li_0 = pybamm.Parameter(
            "Initial plated lithium concentration [mol.m-3]"
        )

Could you pls tell me the defference between "Initial plated lithium concentration [mol.m-3]" and "Typical plated lithium concentration [mol.m-3]"?
Why we use "Typical plated lithium concentration [mol.m-3]" as the defination of scale in lithium plating but not Initial plated lithium concentration?

Thank you so much.

**Comment by brosaplanella** [comment]
The typical one is only used to scale the variable so the solver behaves better. As long as it is within a reasonable range, its exact value should not impact the solution you get.

The initial concentration is used as the initial condition of the systrem.

**Reply by loveinSH** [reply]
Thank you so much！
